dashboard/app/models/regional_partner.rb

```diff
@@ -99,6 +99,11 @@ def summer_workshops_application_state
     end
   end
 
+  # If there is a priority deadline date and it is still upcoming, then return it.  Otherwise return nil.
```

Comment:
is this actually checking whether it is still upcoming?

Reply:
Great catch.